docs/operators/droplet-quickstart.md

```diff
@@ -0,0 +1,227 @@
+# Quickstart for running nwaku on a DigitalOcean Droplet
+
+This guide explains how to run a nwaku node on a 
+DigitalOcean Droplet. We enable the following protocols -
+
+1. Relay
+2. Store
+3. DNS Discovery
+4. Discv5
+
```

Comment:
It may sound extremely obvious, but for completeness, perhaps we can add a couple of sentences here for completeness:
1. One sentence explainer of what a droplet is (basically a "cloud hosted virtual machine") and why you may want to use it to run nwaku (i.e. so that it can run long term without you having to own and maintain the hardware). We have to assume at least some extreme beginners running nwaku.
2. Mention that a prerequisite here is a DigitalOcean account, that running droplets cost money, but that new accounts get 60-days (?) worth of credits (I think :) )
3. Sentence to say that the steps below uses the `doctl` command line utility and that an alternative is using the web interface for creating droplets.

Reply:
Addressed in [3a5e3a6](https://github.com/status-im/nwaku/pull/1133/commits/3a5e3a616e1d169622f28025fb4549cb7c76c255)